adyenv6core/src/com/adyen/v6/forms/AddressForm.java

```diff
@@ -0,0 +1,158 @@
+package com.adyen.v6.forms;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+/**
+ *
+ */
+public class AddressForm {
+    private String addressId;
+    private String titleCode;
+    private String firstName;
+    private String lastName;
+    private String line1;
+    private String line2;
+    private String townCity;
+    private String regionIso;
+    private String postcode;
+    private String countryIsoCode;
+    private String phoneNumber;
+
+
+    @NotNull(message = "{address.title.invalid}")
+    @Size(min = 1, max = 255, message = "{address.title.invalid}")
+    public String getTitleCode() {
+        return titleCode;
+    }
+
+    public void setTitleCode(final String titleCode) {
+        this.titleCode = titleCode;
+    }
+
+    @NotNull(message = "{address.firstName.invalid}")
+    @Size(min = 1, max = 255, message = "{address.firstName.invalid}")
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(final String firstName) {
+        this.firstName = firstName;
+    }
+
+    @NotNull(message = "{address.lastName.invalid}")
```

Comment:
should this be **address.surname.invalid**? like it is used here: 
https://github.com/Adyen/adyen-hybris/pull/177/files#diff-5b328b3d0678e78b67936c813e458208R16

Reply:
this is to provide error message in case surname is null.